qtt/algorithms/markov_chain.py

```diff
@@ -0,0 +1,241 @@
+# -*- coding: utf-8 -*-
+""" Class to generate signals with continous-time Markov chains
+
```

Comment:
Theoretical model describing electron tunneling from and towards dots, for example RTS and Elzermann readout.